test/controllers/test_password_expired_controller.rb

```diff
@@ -16,6 +16,8 @@ class Devise::PasswordExpiredControllerTest < ActionController::TestCase
       confirmed_at: 5.months.ago
     )
     assert @user.valid?
+    assert @user.need_change_password?
```

Comment:
I think this was added while hunting down the problem. Seems reasonable to keep.

Reply:
This was mostly just assuring that our assumptions were being met.